docs/examples/use_cases/pytorch/resnet50/main.py

```diff
@@ -90,55 +90,54 @@ def to_python_float(t):
     else:
         return t[0]
 
-def create_dali_pipeline(batch_size, num_threads, device_id, data_dir, crop, size,
-                       shard_id, num_shards, dali_cpu=False, is_training=True):
-    pipeline = Pipeline(batch_size, num_threads, device_id, seed=12 + device_id)
-    with pipeline:
-        images, labels = fn.file_reader(file_root=data_dir,
-                                        shard_id=args.local_rank,
-                                        num_shards=args.world_size,
-                                        random_shuffle=is_training,
-                                        pad_last_batch=True,
-                                        name="Reader")
-        dali_device = 'cpu' if dali_cpu else 'gpu'
-        decoder_device = 'cpu' if dali_cpu else 'mixed'
-        device_memory_padding = 211025920 if decoder_device == 'mixed' else 0
-        host_memory_padding = 140544512 if decoder_device == 'mixed' else 0
-        if is_training:
-            images = fn.image_decoder_random_crop(images,
-                                                  device=decoder_device, output_type=types.RGB,
-                                                  device_memory_padding=device_memory_padding,
-                                                  host_memory_padding=host_memory_padding,
-                                                  random_aspect_ratio=[0.8, 1.25],
-                                                  random_area=[0.1, 1.0],
-                                                  num_attempts=100)
-            images = fn.resize(images,
-                               device=dali_device,
-                               resize_x=crop,
-                               resize_y=crop,
-                               interp_type=types.INTERP_TRIANGULAR)
-            mirror = fn.random.coin_flip(probability=0.5)
-        else:
-            images = fn.image_decoder(images,
-                                      device=decoder_device,
-                                      output_type=types.RGB)
-            images = fn.resize(images,
-                               device=dali_device,
-                               size=size,
-                               mode="not_smaller",
-                               interp_type=types.INTERP_TRIANGULAR)
-            mirror = False
-
-        images = fn.crop_mirror_normalize(images.gpu(),
-                                          dtype=types.FLOAT,
-                                          output_layout="CHW",
-                                          crop=(crop, crop),
-                                          mean=[0.485 * 255,0.456 * 255,0.406 * 255],
-                                          std=[0.229 * 255,0.224 * 255,0.225 * 255],
-                                          mirror=mirror)
-        labels = labels.gpu()
-        pipeline.set_outputs(images, labels)
-    return pipeline
+
+@pipeline
```

Comment:
If we decide to go ahead with changing the examples, I'd use here the whole name:
```@nvidia.dali.pipeline.pipeline_def``` or ```@nvidia.dali.pipeline.pipeline```

Reply:
In the examples we use `Pipeline` instead of `nvidia.dali.pipeline.Pipeline`, so I believe it's better to use `pipeline_def` instead of `nvidia.dali.pipeline.pipeline_def`